paper/paper.md

```diff
@@ -0,0 +1,103 @@
+
+---
+title: 'MoSDeF-dihedral-fit: A simple software package to fit dihedrals via the MoSDeF software'
+
+tags:
+  - Python
+  - Molecular simulations
+  - Molecular mechanics
+  - Molecular dynamics
+  - Monte Carlo
+  - Quantum mechanics
+  - dihedral fitting
+  - torsion fitting
+  - force field
+  - MoSDeF
+  - GOMC
+  - MoSDeF-GOMC
+
+authors:
+  - name: Brad Crawford
+    orcid: 0000-0003-0638-7333
+    equal-contrib: true
+    affiliation: "1, 2"
+  - name: Co D. Quach
+    orcid:
+    affiliation: "3, 4"
+  - name: Nicholas C. Craven
+    orcid:
+    affiliation: "4, 5"
+  - name: Christopher R. Iacovella
+    orcid:
```

Comment:
I couldn't find Chris's ORCID, tried google but there wasn't any hit so he may or may not have one.